spec/lib/tasks/populate_postcodes_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rake"
```

Comment:
Nitpick: I'd rename this file to be consistent with the rake task file, i.e. `populate_postcodes_table_spec.rb`.